front/app/components/ProjectAndFolderCards/components/ProjectFolderCard/index.tsx

```diff
@@ -276,6 +278,78 @@ const FolderDescription = styled.div`
   margin-top: 15px;
 `;
 
+const ContentFooter = styled.div`
```

Comment:
I'm not  fan of all these styled components, but this is exactly what the Project Card uses, and I wanted to maintain the same style + behaviour.